README.md

```diff
@@ -1,70 +1,66 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
 
-## Available Scripts
+> Math Magicians
```

Comment:
- [ ] **Required:** KINDLY update the project title from `Project Name` to `Math Magicians` or similar. Also, update the description to include one or more sentences that explain what the project entails.